website/docs/client/java/partials/java_transport.md

```diff
@@ -774,20 +774,20 @@ openlineage.transport.credentialsFile=path/to/credentials.json
 
 ```java
 import io.openlineage.client.OpenLineageClient;
-import io.openlineage.client.transports.dataplex.DataplexConfig;
-import io.openlineage.client.transports.dataplex.DataplexTransport;
+import io.openlineage.client.transports.gcplineage.GcpLineageTransportConfig;
+import io.openlineage.client.transports.dataplex.GcpLineageTransport;
 
 
-DataplexConfig dataplexConfig = new DataplexConfig();
+GcpLineageTransportConfig gcplineageConfig = new GcpLineageTransportConfig();
 
-dataplexConfig.setProjectId("your_kinesis_stream_name");
-dataplexConfig.setLocation("your_aws_region");
-dataplexConfig.setMode("sync");
-dataplexConfig.setCredentialsFile("path/to/credentials.json");
+gcplineageConfig.setProjectId("your_kinesis_stream_name");
```

Comment:
do you want to change the project id example here?

Reply:
done